x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/bulk_create_attachments.ts

```diff
@@ -218,17 +210,6 @@ export default ({ getService }: FtrProviderContext): void => {
           });
         });
 
-        it('should bulk create 100 file attachments when there is another attachment type in the request', async () => {
```

Comment:
Do you remember why we had the test in the first place?

Reply:
Not really 😄 @jonathan-buttner ?

Reply:
Yeah so I think the file attachment code limits how many files can be attached to a case to 100, what this test was covering is that the code _**only**_ counts file attachments and would ignore a different type of attachment when doing the counting.

Maybe we don't need it anymore if we're limiting the total attachments in a single request to 100 🤷‍♂️ 

https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/server/common/limiter_checker/limiters/files.ts#L15

I think we have some unit test coverage there too so we're probably ok.

Reply:
Thanks, Jonathan. It seems it is covered by the unit tests.